reference/test/stop/combined.cpp

```diff
@@ -105,14 +119,14 @@ TEST_F(Combined, WaitsTillIteration)
  * the very small time picked and huge iteration count. */
 TEST_F(Combined, WaitsTillTime)
 {
-    constexpr double timelimit = 1.0e-9;
     constexpr int testiters = 10;
+    constexpr int timelimit = 10;
```

Comment:
nit: please add the time unit, so it is clear that `10` is `10 ms`, and not `10 s`
```suggestion
    constexpr int timelimit_ms = 10;
```